docs/guides/http_crawler.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: http-crawler-guide
+title: "HttpCrawler guide"
+sidebar_label: "HttpCrawler"
+description: Your first steps into the world of scraping with Crawlee
+---
+
+import ApiLink from '@site/src/components/ApiLink';
```

Comment:
I would start with creating a guide for the high-level ones only, but if the content is already written, I guess it would be a shame not to use it.

Reply:
The content was written via copy & paste :P I agree with @B4nan - this may be too abstract novice users, as they probably want something that "just works". `HttpCrawler` is still visible in API so more advanced people still can get to it.

I'll remove this, we can re-add this if there's a need to.

> i though the class will be abstract and we will document only its children

The point of making this non-abstract is that people can attach their own layers on top of this. They can use other XML parsers, or image transformation tools and so on.